pages/01_metadata.py

```diff
@@ -0,0 +1,156 @@
+# Define the page's content within a variable called layout or
+# a function called layout that returns the content.
+#
+# File upload component:
+# https://dash.plotly.com/dash-core-components/upload
+
+import base64
+import os
+
+import dash
+import dash_bootstrap_components as dbc
+import pandas as pd
+import yaml
+from dash import Input, Output, State, callback, dcc, html
+
+###############
+# Register this page
+dash.register_page(__name__)
+
+
+####################
+# Read yamls in dir as pandas dataframe
+
+
+def df_from_metadata_yaml_files(parent_dir):
+    # TODO: refactor, I think it could be more compact (see below)
+
+    list_metadata_files = [
+        el for el in os.listdir(parent_dir) if el.endswith("metadata.yaml")
+    ]
+
+    list_df_metadata = []
+    for yl in list_metadata_files:
+        with open(os.path.join(parent_dir, yl)) as f:
```

Comment:
Instead of `os.path.join` consider using `pathlib`. It makes paths more compact and readable, and it has wider functionality compared to `os.path`. That's what we use by default in the group.

Reply:
example usage:
```python
import pathlib as pl

parent_dir = pl.Path("This/is/My/Folder")
child_file = parent_dir / "filename"
```

Reply:
complied, thanks!